bentoml/server/bento_api_server.py

```diff
@@ -194,21 +193,66 @@ def bento_service_api_wrapper(api, service_name, service_version):
     summary_name = str(service_name) + "_" + str(api.name)
     request_metric_time = Summary(summary_name, summary_name + " request latency")
 
+    def log_image(request, request_id):
+        img_prefix = 'image/'
+        log_folder = config('logging').get('base_log_folder')
+
+        all_paths = []
+
+        if request.content_type.startswith(img_prefix):
+            filename = '{request_id}-{timestamp}.{ext}'.format(
+                request_id=request_id,
+                timestamp=int(time.time()),
+                ext=request.content_type[len(img_prefix) :],
+            )
+            path = os.path.join(log_folder, filename)
+            all_paths.append(path)
+            with open(path, 'w') as f:
+                f.write(request.get_data())
+
+        for name in request.files:
+            file = request.files[name]
+            if file and file.filename:
+                orig_filename = secure_filename(file.filename)
+                filename = '{request_id}-{timestamp}-{orig_filename}'.format(
```

Comment:
consider '{timestamp}-{request_id}-{orig_filename}'? so when doing manual debugging, image files will be sorted nicely by time?